src/main/java/com/exasol/config/ClusterConfiguration.java

```diff
@@ -38,7 +38,14 @@ public ClusterConfiguration(final Map<String, String> parameters) {
     public String getAuthenticationToken() {
         return this.parameters.get(GLOBAL_SECTION + "/AuthenticationToken");
     }
-
+    /**
+     * Get the database version from the Global section.
```

Comment:
```suggestion
     * Get the database version.
```

too much information ;-)